tests/acceptance/features/webUISharingFolderAdvancedPermissionMultipleUsers/sharedFolderWithMultipleUsersAdvancedPermissions.feature

```diff
@@ -60,9 +60,10 @@ Feature: Sharing folders with multiple internal users using advanced permissions
       | Advanced permissions | Advanced permissions | update            | update                | read, update         |
       | Advanced permissions | Advanced permissions | delete, update    | delete, update        | read, delete, update |
       | Advanced permissions | Advanced permissions | update, create    | update, create        | read, update, create |
+      | Advanced permissions | Advanced permissions | create            | create                | read, create         |
+      | Advanced permissions | Advanced permissions | delete, create    | delete, create        | read, delete, create |
+
 
-  @issue-ocis-717
-  #after fixing the issue merge this scenario into the one above
```

Comment:
If this has been merged into the one above, can we drop it?

Reply:
yes, actually we could and adjust the expected failures files accordingly. THX

Reply:
@LukasHirt 
done, please review again